bigbluebutton-html5/imports/ui/core/graphql/queries/users.ts

```diff
@@ -38,6 +38,7 @@ subscription UserListSubscription($offset: Int!, $limit: Int!) {
       joined
       listenOnly
       voiceUserId
+      muted
```

Comment:
This prop was removed on purpose!
The idea is reduce the traffic flow of this subscription every time someone mute/unmute.

Look at https://github.com/bigbluebutton/bigbluebutton/pull/20618 to see the new way to get mute/unmute status.